README.md

```diff
@@ -82,32 +102,37 @@ This module assumes your change log is a [markdown](http://daringfireball.net/pr
 A cool description (optional).
 
 ## unreleased
-* foo
+
+- foo
```

Comment:
please don't change the example changelog template

Reply:
my bad, formatOnSave did that.